hello_world/react-client/src/components/Faucet.js

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import { APIClient } from '@liskhq/lisk-api-client';
+import accounts from '../accounts.json';
+import{ TransferTransaction, utils } from '@liskhq/lisk-transactions';
+import * as cryptography from '@liskhq/lisk-cryptography';
+
+const networkIdentifier = cryptography.getNetworkIdentifier(
+    "23ce0366ef0a14a91e5fd4b1591fc880ffbef9d988ff8bebf8f3666b0c09597d",
+    "Lisk",
+);
+
+const API_BASEURL = 'http://localhost:4000';
+
+const api = new APIClient([API_BASEURL]);
+
+const dateToLiskEpochTimestamp = date => (
+    Math.floor(new Date(date).getTime() / 1000) - Math.floor(new Date(Date.UTC(2016, 4, 24, 17, 0, 0, 0)).getTime() / 1000)
```

Comment:
I think there is something in lisk-elements to handle the epoch (at least a constant)